tools/enhancements/summary.go

```diff
@@ -24,29 +24,54 @@ func UpdateGitRepo() error {
 	return nil
 }
 
-const desiredRefSetting = "+refs/pull/*/head:refs/remotes/origin/pr/*"
+// findOpenShiftRemote looks through the remotes for the current git
+// repository to find one with "openshift/enhancements" in the URL
```

Comment:
I suggest adding a requirement to have an upstream remote in the README.

Reply:
Good idea. I added another commit to this PR to do that.